elastic/tests/test_metrics.py

```diff
@@ -28,7 +28,7 @@
         pytest.param([5, 0, 0], False, 250, id='v5'),
         pytest.param([6, 3, 0], False, 250, id='v6.3.0'),
         pytest.param([7, 2, 0], False, 244, id='v7.2.0'),
-        pytest.param([8, 6, 2], False, 249, id='v8.6.0'),
+        pytest.param([8, 6, 2], False, 252, id='v8.6.0'),
```

Comment:
It would be nice to have some more user-facing tests for this. For instance, given an ES8 instance make sure we do in fact collect these metrics from it.